farmdata2_modules/fd2_tabs/fd2_barn_kit/transplantingReport.html

```diff
@@ -0,0 +1,639 @@
+                endDate: dayjs().format('YYYY-MM-DD').toString(),
+
+                stillLoading: true,
+                reportVisible: false,
+                transplantingLogs: [],
+                updateLog: [],
+
+                sessionToken: null,
+                selectedCrop: 'All',
+                selectedArea: 'All',
+
+                headers: ['Date', 'Crop', 'Area', 'Bed Feet', 'Row Feet', 'Rows/Bed', 'Trays', 'Workers', 'Hours','Comments','User'],
+                visibleColumns: [true, true, true, true, true, true, true, true, true, true, true],
+
+                idToCropMap: new Map(),
+                cropToIDMap: new Map(),
+                idToUserMap: new Map(),
+                userToIDMap: new Map(),
+                areaToIDMap: new Map(),
+                unitToIDMap: new Map(),
+                configToIDMap: new Map(),
+
+                rowBeingEdited: false,
+                errBanner: null,
+            },
+            methods: {
+                hide(){
+                    this.reportVisible = false
+                    this.transplantingLogs = []
+                },
+                startDateChange(selectedDate){
+                    this.startDate = selectedDate
+                    //this.reportVisible = false
+                    //this.transplantingLogs = []
+                },
+                endDateChange(selectedDate){
+                    this.endDate = selectedDate
+                    //this.reportVisible = false
+                    //this.transplantingLogs = []
+                },
+                seedingLogRequestWithDates(){
+                    this.reportVisible = true
+                    let link = '/log.json?type=farm_transplanting&timestamp[ge]=' + dayjs(this.startDate).unix() + '&timestamp[le]=' + dayjs(this.endDate).unix()
+
+                    getAllPages(link, this.transplantingLogs)
+                        .then(() => {
+                            this.stillLoading = false
+                        })
+                        .catch((err) => { 
+                            this.errBanner = true
+                            var errorBanner = document.getElementById('scrollBanners')
+                            var pageHeader = document.getElementById('navbar')
+                            var headerBounds = pageHeader.getBoundingClientRect()
+                            var alertBounds = errorBanner.getBoundingClientRect()
+                            scrollBy({
+                                top: alertBounds.top - headerBounds.bottom,
+                                behavior: 'smooth'
+                            })
+                        })
+                    this.stillLoading = true
+                },
+                cropChange(selectedCrop){
+                    this.selectedCrop = selectedCrop
+                },
+                areaChange(selectedArea){
+                    this.selectedArea = selectedArea
+                },
+                configChange(){
```

Comment:
This function does not seem to be used at all.  Is it here for a reason?  Could it be a leftover from something that was tried in the SeedingReport and then removed?

Reply:
This is a leftover from the Seeding Report. This is actually still present in the Seeding Report [[here](https://github.com/DickinsonCollege/FarmData2/blob/main/farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.html#L182)]. Is this here for when the seeding is changed labor is appropriately handled? @JingyuMarcelLee